wpeverest/user-registration
# Deactivation popup

### All Submissions:






### Changes proposed in this Pull Request:



Enhancement of deactivation popup. This PR will enhance deactivation popup.

### How to test the changes in this Pull Request:

1. First go to Install & activate the plugin
2. Now click on deactivate the plugin 
3. Check if the deactivation popup appears or not and check whether the submitted data is going to the stats server. 

### Types of changes:




### Other information:




### Changelog entry

> Enhancement - Deactivation popup redesign
